### PR TITLE
Pickup from henne's #5253

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

**Comment by hennevogel** [comment]
?

**Comment by coolo** [comment]
just trying to distract myself from the news :)

And I need these fixes for the chromium PR and the cassette one, so I wanted to help out

File: .circleci/config.yml [hunk 1]
```diff
@@ -170,7 +170,7 @@ jobs:
       - store_artifacts:
           path: ./src/api/log/
           destination: minitest
-  backend:
+  backend_test:
```
Comment (hennevogel) [inline .circleci/config.yml]: Why do you rename this?

Comment (coolo) [reply .circleci/config.yml]: See the commit log - to free the namespace

Comment (coolo) [reply .circleci/config.yml]: (the story is that we need the container to be named 'backend' so that the cassettes will work)

Comment (hennevogel) [reply .circleci/config.yml]: If we are going down that route you mean... :-)

Comment (coolo) [reply .circleci/config.yml]: well, let me put it this way: we call the other jobs minitest and rspec, not api and webui.

**Comment by coolo** [comment]
but it seems there is a problem with it - ci/circleci: backend is expected. Is there some more config than what is in git? @ChrisBr ?

**Comment by coolo** [comment]
https://circleci.com/workflow-run/edad5b58-ebe1-4db5-a087-a4a19af699f0 doesn't mention backend

**Comment by ChrisBr** [comment]
> but it seems there is a problem with it - ci/circleci: backend is expected. Is there some more config than what is in git?

What problem? I'm lost :thinking:

**Comment by coolo** [comment]
see circle checks:
ci/circleci: backend Expected — Waiting for status to be reported

File: .circleci/config.yml (hunk 1 above)
Comment (hennevogel) [reply .circleci/config.yml]: I don't care. This is unrelated. Drop it please

**Comment by hennevogel** [comment]
>  ci/circleci: backend Expected — Waiting for status to be reported

This is the github branch protection

**Comment by coolo** [comment]
means it will go away once it's merged to master?

**Comment by hennevogel** [comment]
No it means someone with admin permissions would have to change it.

File: .circleci/config.yml (hunk 1 above)
Comment (coolo) [reply .circleci/config.yml]: Na gut, weil Du es bist

**Comment by hennevogel** [comment]
@openSUSE/obs-frontend can someone else please review this please?

File: src/api/app/models/project.rb [hunk 2]
```diff
@@ -609,7 +609,7 @@ def write_to_backend
     reset_cache
 
     if CONFIG['global_write_through'] && !@commit_opts[:no_backend_write]
-      login = @commit_opts[:login] || User.current.login
+      login = @commit_opts[:login] || User.current_login
```
Comment (ChrisBr) [inline src/api/app/models/project.rb]: How is this related to the commit message?

File: src/api/spec/features/webui/users/users_home_project_spec.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 require 'browser_helper'
 
 RSpec.feature "User's home project creation", type: :feature, js: true do
-  let!(:user) { create(:confirmed_user, login: 'moi') }
+  let!(:user) { create(:confirmed_user, login: 'hans') }
```
Comment (ChrisBr) [inline src/api/spec/features/webui/users/users_home_project_spec.rb]: Why?

File: src/api/spec/features/webui/image_templates_spec.rb [hunk 4]
```diff
@@ -1,7 +1,4 @@
 require 'browser_helper'
-# WARNING: This test require real backend answers for projects/packages, make
```
Comment (ChrisBr) [inline src/api/spec/features/webui/image_templates_spec.rb]: Are these all? I think in the meantime we got more of these ...

**Comment by ChrisBr** [comment]
Code wise it LGTM. However, 78069f707336bd8f0884c099000e01090a5c51f4 lacks a commit message. I have no idea what this commit is supposed to fix. Also I think that it does several (unrelated) things at once which would be nice to either reflect in the commit message and/or split in smaller commits.

File: src/api/spec/features/webui/image_templates_spec.rb (hunk 4 above)
Comment (coolo) [reply src/api/spec/features/webui/image_templates_spec.rb]: yeah, but henne only wanted to remove them from features tests. I'll remove the rest later on :)

Comment (ChrisBr) [reply src/api/spec/features/webui/image_templates_spec.rb]: Fine for me

**Comment by coolo** [comment]
as 5253 is reopened, let's close this one - I demonstrated that the rubocop fixes don't break anything
